src/lib/components/code-block.svelte

```diff
@@ -27,7 +27,7 @@
 </script>
 
 {#if inline}
-  <code class="language-json text-xs">
+  <code class="language-json" style="white-space: nowrap;">
```

Comment:
Break word may be better

Reply:
I literally never want it to wrap. I need the height to stay consistent.